api/types/mount/mount.go

```diff
@@ -1,24 +1,30 @@
 package mount
 
+import (
+	"os"
+)
+
 // Type represents the type of a mount.
 type Type string
 
+// Type constants
 const (
-	// TypeBind BIND
-	TypeBind Type = "bind"
-	// TypeVolume VOLUME
+	TypeBind   Type = "bind"
```

Comment:
Rather than remove documentation, let's actually add it in. You can copy from swarmkit.